feat(wallet): create company wallet page

manojava-gk opened the pull request:
## Description

Company wallet listing page along with api integration

## Why

new feature

## Issue

https://github.com/eclipse-tractusx/portal-frontend/issues/443

## Checklist

- [x] I have followed the [contributing guidelines](https://github.com/eclipse-tractusx/portal-assets/blob/main/developer/Technical%20Documentation/Dev%20Process/How%20to%20contribute.md#commit-and-pr-guidelines)
- [x] I have performed a self-review of my own code
- [x] I have successfully tested my changes locally

manojava-gk commented:
FYI - Zero linter issues in newly created folders

manojava-gk commented on src/types/Constants.ts:
```diff
@@ -213,6 +214,7 @@ export enum ROLES {
   SUBMITTED_APPLICATION = 'view_submitted_applications',
   REQUEST_SSICREDENTIAL = 'request_ssicredential',
   DECISION_SSICREDENTIAL = 'decision_ssicredential',
+  COMPANY_WALLET = 'CX Admin',
```
@jjeroch Please provide role info

jjeroch reviewed (changes requested):
- Grouping update needed
  - Group 1: "Frameworks" shows all credentials with credential type "Framework.*" 
  - Group 2: "Company Role" shows all credentials with credential type "CompanyRole.*"
  - Group 3: "Others" shows all other credentials which are responded
- where the BPNL is currently displayed, there the Credential Type should be shown (big and bold)
- Where the current credential type is displayed, there the BPNL should show up (small and not bold)
- Please mention "Expiry:" in front of the expiry date

manojava-gk commented:
> * Grouping update needed
>   
>   * Group 1: "Frameworks" shows all credentials with credential type "Framework.*"
>   * Group 2: "Company Role" shows all credentials with credential type "CompanyRole.*"
>   * Group 3: "Others" shows all other credentials which are responded
> * where the BPNL is currently displayed, there the Credential Type should be shown (big and bold)
> * Where the current credential type is displayed, there the BPNL should show up (small and not bold)
> * Please mention "Expiry:" in front of the expiry date

Addressed the comments. but waiting for appropriate data to test @jjeroch

jjeroch commented on src/assets/locales/de/main.json:
```diff
@@ -1675,6 +1676,21 @@
+        "button2": "Request new Company Credential",
+        "button3": "Get more details of VCs (coming soon)"
+      },
+      "expiry": "Expiry: "
```
German Translation? ;)

manojava-gk replied:
Check now
